tools/lint/rules.py

```diff
@@ -414,6 +414,11 @@ class WebPlatformTestRegexp(Regexp):
     pattern = br"web\-platform\.test"
     name = "WEB-PLATFORM.TEST"
     description = "Internal web-platform.test domain used"
+    to_fix = """
+        use [server-side substitution](https://wptserve.readthedocs.io/en/latest/pipes.html#sub),
```

Comment:
```suggestion
        use [server-side substitution](https://web-platform-tests.org/writing-tests/server-pipes.html#sub),
```

I don't think our `readthedocs` is regularly updated anymore. (Maybe we should just purge it? cc @jgraham )

Reply:
Ah thanks, I got the link from https://web-platform-tests.org/writing-tests/file-names.html#test-features actually. Let me update both this and that docs/ link.